.github/workflows/ci.yml

```
@@ -66,6 +66,32 @@ jobs:
       - name: Run YAML lint
         run: pnpm lint:yaml
 
+      - name: Validate Grafana dashboards JSON
+        run: |
+          echo "Validating Grafana dashboard JSON files..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)